### PR TITLE
Fix generate rb key

### PR DESCRIPTION
**Pull request by pigletfly**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by pigletfly** [comment]
@Garrybest

**Comment by Garrybest** [comment]
See #1003, still fix the compatible effects.

File: pkg/util/names/names_test.go [hunk 1]
```diff
@@ -76,3 +76,27 @@ func TestGetClusterName(t *testing.T) {
 		})
 	}
 }
+
+func TestGenerateBindingReferenceKey(t *testing.T) {
+	tests := []struct {
+		name      string
+		namespace string
```
Comment (RainbowMango) [inline pkg/util/names/names_test.go:83]:
```suggestion
		want string
```
We can add a `want` field here.

Comment (RainbowMango) [reply pkg/util/names/names_test.go:83]: Sorry, not replace `namespace` with `want`, but add a new field.

**Comment by RainbowMango** [comment]
Hi @pigletfly 
We are going to cut a new release this morning, could you please rebase this PR? I want this unit test could be included.

**Comment by pigletfly** [comment]
rebased

**Comment by RainbowMango** [comment]
/lgtm

There is something wrong with FOSSA, I'll re-trigger it once it gets fixed.

**Pull request merged**